src/main/default/lwc/provideDetailsScreen/provideDetailsScreen.js

```
@@ -138,17 +147,22 @@ export default class ProvideDetailsScreen extends LightningElement {
     const priorityInput = this.template.querySelector(
       'c-quantic-case-classification[title="priority"]'
     );
-    const originInput = this.template.querySelector(
-      'c-quantic-case-classification[title="origin"]'
+    const reasonInput = this.template.querySelector(
+      'c-quantic-case-classification[title="reason"]'
+    );
+    const typeInput = this.template.querySelector(
+      'c-quantic-case-classification[title="type"]'
     );
-    return { priorityInput, originInput };
+    return { priorityInput, reasonInput, typeInput };
   }
 
   extractDataFromSessionStorage() {
-    if (sessionStorage.previousNavigation && sessionStorage.caseData) {
-      const sessionStorageObject = JSON.parse(sessionStorage.caseData);
-      this.priority = sessionStorageObject.priority;
-      this.origin = sessionStorageObject.origin;
+    if (!sessionStorage.valuesUpdated && sessionStorage.caseData) {
+      try {
+        this.sessionStorageCaseObject = JSON.parse(sessionStorage.caseData);
+      } catch (err) {
```

Please log a warning. Suppressing errors makes troubleshooting quite difficult.

---

All warnings added👌